bin/build-env.sh

```diff
@@ -0,0 +1,30 @@
+. .env
```

Comment:
Add a shebang to those scripts, cause when running from `zsh` it's changing the working directory.